packages/web-components/fast-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,217 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowLeft,
+    keyCodeArrowRight,
+    keyCodeArrowUp,
+    keyCodeEnter,
+} from "@microsoft/fast-web-utilities";
+import { RadioControl } from "../radio";
+
+export class RadioGroup extends FASTElement {
+    @attr({ attribute: "readonly", mode: "boolean" })
+    public readOnly: boolean;
+    private readOnlyChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLInputElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("readonly", "");
+                } else {
```

Comment:
FYI - you generally just set the underlying *property* that the attribute is mapped to:
```
radio.readOnly = true;
// vs
radio.setAttribute('readonly', '');
```

Not only is it less verbose, but it is also generally significantly faster: https://jsperf.com/get-set-attribute-vs-direct https://jsbench.me/dbk9sookk1/1

Reply:
Ok yea, Since it was typed HTMLInputElement that slipped my mind as a possibility, will fix

Reply:
also I thought I was adding the attribute and not setting it to true which further confused things.

Reply:
I believe all of these are now updated to the more efficient method